ocfweb/account/recommender.py

```diff
@@ -0,0 +1,24 @@
+from ocflib.account.creation import validate_username

```

Comment:
This whole file is unused and duplicates https://github.com/ocf/ocflib/pull/102, though there I suggested that the code be permanently moved here. (Oops, I should have left my comments here instead.)

Reply:
![screenshot-supernova ocf berkeley edu-8684-2017-11-17-09-39-37-165](https://user-images.githubusercontent.com/16551944/32960829-421ef38a-cb7b-11e7-8373-13a136975e17.png)
![screenshot-supernova ocf berkeley edu-8684-2017-11-17-09-39-12-874](https://user-images.githubusercontent.com/16551944/32960830-425f315c-cb7b-11e7-8d25-5815353efd81.png)
This verify text and green/red highlighting occurs after a specified timeout after user has stopped timing (currently 250ms). 
I originally had the recommendations immediately below the username field (as opposed to the sidebar) but if they were listed vertically, they took up too much space, and I didn't like the way they looked when listed horizontally. But this cosmetic judgment is up to a majority consensus.